website/source/docs/commands/connect/envoy.html.md.erb

```diff
@@ -94,6 +79,54 @@ allowed to access by [Connect intentions](/docs/connect/intentions.html).
    always specifies `--config-file` and `--v2-config-only` and by default passes
    `--disable-hot-restart` see [hot restart](#hot-restart).
 
+#### Envoy Sidecar Proxy Options
+
+* `-sidecar-for` - The _ID_ (not name if they differ) of the service instance
+  this proxy will represent. The target service doesn't need to exist on the
+  local agent yet but a [sidecar proxy
+  registration](/docs/connect/registration/service-registration.html) with
+  `proxy.destination_service_id` equal to the passed value must be present. If
+  multiple proxy registrations targeting the same local service instance are
+  present the command will error and `-proxy-id` should be used instead.
+
+-> **Note:** If ACLs are enabled, a token granting `service:write` for the
```

Comment:
I think if you tab this in it will nest under the bullet that it refers to.

Reply:
This note not specific to the other bullets but a note about the sidecar proxy mode of operation of this command.